src/sql/workbench/contrib/notebooksExplorer/browser/media/book.svg

```diff
@@ -0,0 +1,7 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" viewBox="0 0 16 16">
```

Comment:
These should probably just be added to the common icons - they seem like something we're likely to use elsewhere https://github.com/microsoft/azuredatastudio/blob/master/src/sql/media/icons/common-icons.css#L1

Reply:
@MaddyDev Are you going to add this to the common icons?

Reply:
I looked at it yesterday and wanted to finalize with PMs about these icons before moving them to common icons.